pkg/project/resource_project_repo_test.go

```diff
@@ -114,3 +114,128 @@ func TestAccProjectRepo(t *testing.T) {
 		},
 	})
 }
+
+/*
+Test to assign large number of repositories to a project
+*/
+func TestAccAssignMultipleReposInProject(t *testing.T) {
+
+	const numRepos = 100
+	const repoNameInitial = "repo-"
+
+	name := "tftestprojects" + randSeq(10)
+	resourceName := "project." + name
+	projectKey := strings.ToLower(randSeq(6))
+
+	repoNamesStr := func(repoCount int) string {
+		var repoNames []string
+		for i := 0; i < repoCount; i++ {
+			repoNames = append(repoNames, repoNameInitial+strconv.Itoa(i))
```

Comment:
Are you trying to create JSON with an array????Like, I have no clue what you're doing here

Reply:
`json.Marshal` + `fmt.Printf` will do that job much clearer

Reply:
improved